hie-docs-generator/src/Options.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Options where
+
+import qualified Data.Map as Map
+import           Haskell.Ide.ApplyRefactPlugin
+import           Haskell.Ide.Engine.BasePlugin
+import           Haskell.Ide.Engine.PluginDescriptor
+import           Haskell.Ide.ExamplePlugin2
+import           Haskell.Ide.ExamplePluginAsync
+import           Haskell.Ide.GhcModPlugin
+import           Haskell.Ide.HaRePlugin
+import           Options.Applicative
+
+data Config =
+  Config {plugins :: Plugins
+         ,prefix :: FilePath}
+
+config :: Parser Config
+config =
+  Config pluginList <$>
+  strOption (long "prefix" <> metavar "PREFIX" <>
+             help "directory where the docs are written to (defaults to current directory)")
+
+pluginList :: Plugins
```

Comment:
We should try to make this common with the one in the app/Main.hs, else we have to change it in two places